patches/third_party-blink-renderer-core-frame-navigator.idl.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/core/frame/navigator.idl b/third_party/blink/renderer/core/frame/navigator.idl
+index f13febb148e9..b8e0bbc2b66e 100644
+--- a/third_party/blink/renderer/core/frame/navigator.idl
++++ b/third_party/blink/renderer/core/frame/navigator.idl
+@@ -36,7 +36,6 @@
+ 
+ Navigator includes NavigatorConcurrentHardware;
+ Navigator includes NavigatorCookies;
+-Navigator includes NavigatorDeviceMemory;
```

Comment:
this appears to be sufficient, but can you comment it out instead `// Navigator includes NavigatorDeviceMemory;`?

Reply:
Done.